pkg/volume/volume.go

```diff
@@ -132,11 +132,39 @@ type Attacher interface {
 	Attach() error
 }
 
+type PhysicalAttacher interface {
+	// WaitForAttach examines the file system for the device; if
```

Comment:
Don't suggest implementation in the GoDoc for an interface method. Meaning don't say *how* this method should be implemented (e.g. "examines the file system"), instead focus on *what* this method should do (e.g. "WaitForAttach will block until the volume has been successfully attached to this machine. If the volume is not attached within X minutes, it will return an error").